ingestion_tools/scripts/tests/fixtures/dataset1.yaml

```diff
@@ -23,6 +23,11 @@ annotations:
     file_format: csv
     glob_string: particle_lists/{run_name}_fas.csv
     shape: Point
+collection_metadata:
```

Comment:
is the code for collection_metadata support not part of this PR?

Reply:
Good catch! the collection_metadata test file got missed in being included.